src/core_scan/Core_scan.ml

```diff
@@ -633,7 +633,7 @@ let iter_targets_and_get_matches_and_exn_to_errors config
 (* File targeting and rule filtering *)
 (*****************************************************************************)
 
-let xtarget_of_file ~parsing_cache_dir (xlang : Xlang.t) (file : Fpath.t) :
+let xxxtarget_of_file ~parsing_cache_dir (xlang : Xlang.t) (file : Fpath.t) :
```

Comment:
Why this change?

Reply:
it was to test this PR, that is to test whether the new workflow would report an error because semgrep-pro relies on xtarget_of_file, so renaming this func in semgrep will break semgrep-pro

Reply:
I think he introduced an incompatibility on purpose to make sure the CI check would fail and block the merge.